frontend/src/App.tsx

```diff
@@ -363,11 +404,15 @@ async function GameController(
       townIsPubliclyListed: video.isPubliclyListed,
       emitMovement,
       emitGameInvite,
+      emitInviteResponse,
+      emitFinishGame,
```

Comment:
emitFinishGame is used inside the app() function inside of App.tsx.  I'm a bit wary of moving the emitFinishGame function around to try to give app() access to it without including it in the appstate just because of the structure that's already in place.